tests/raftstore/server.rs

```diff
@@ -147,18 +149,27 @@ impl Simulator for ServerCluster {
         let pd_worker = FutureWorker::new("test-pd-worker");
         let server_cfg = Arc::new(cfg.server.clone());
         let security_mgr = Arc::new(SecurityManager::new(&cfg.security).unwrap());
-        let mut server = Server::new(
-            &server_cfg,
-            &security_mgr,
-            cfg.coprocessor.region_split_size.0 as usize,
-            store.clone(),
-            sim_router.clone(),
-            resolver,
-            snap_mgr.clone(),
-            pd_worker.scheduler(),
-            Some(engines.clone()),
-            Some(import_service),
-        ).unwrap();
+        let mut server = None;
+        for _ in 0..100 {
+            server = Some(Server::new(
+                &server_cfg,
+                &security_mgr,
+                cfg.coprocessor.region_split_size.0 as usize,
+                store.clone(),
+                sim_router.clone(),
+                resolver.clone(),
+                snap_mgr.clone(),
+                pd_worker.scheduler(),
+                Some(engines.clone()),
+                Some(import_service.clone()),
+            ));
+            if let Some(Err(Error::Grpc(GrpcError::BindFail(ref addr, ref port)))) = server {
+                debug!("fail to create a server: bind fail {:?}", (addr, port));
+                thread::sleep(Duration::from_millis(100));
+                continue;
+            }
```

Comment:
If it is not a `BindFail`, it should panic immediately.

Reply:
And I think we can eliminate the `Option`.